packages/packages.go

```diff
@@ -412,12 +426,6 @@ func (f *Filter) legacyApply(ctx context.Context, packages Packages) Packages {
 			continue
 		}
 
-		if f.Capabilities != nil {
-			if valid := p.WorksWithCapabilities(f.Capabilities); !valid {
-				continue
-			}
-		}
-
```

Comment:
Removed from legacy. These clients should not send `capabilities` in their queries.